PlusContent/privacy-mode.md

```diff
@@ -19,13 +19,13 @@ Internally, a privacy enhanced box is based on three defaults:
 - `C:\Windows`
 - `C:\Program Files`
 - `C:\Program Files (x86)`
-- HKLM (but not HKCU) registry keys using `ReadKeyPath=HKEY_LOCAL_MACHINE*`
+- Registry resources under HKLM (but not HKCU) are readable and can be sandboxed
 - **Note:** The read access provides a good balance between privacy and convenience. One could, of course, drill down to identify selected system resources that may leak private data and further restrict them (using `Write[File/Key]Path`) if desired.
 
 2. **Hide (and block access to) user space:**
-- In user space, a privacy box works in **default block** mode: every path is a Write[File/Key]Path (which does not allow read/write changes to sandbox) unless specifically allowed by an over-riding rule. Access to selected paths is enabled by invoking [Rule Specificity](../PlusContent/RuleSpecificity.md).
+- In user space, a privacy box works in **default block** mode: every path is a Write[File/Key]Path. This hides all files and folders outside the sandbox, but allows new files and folders to be created in the sandbox (unless specifically allowed by an over-riding rule). Access to selected paths is enabled by invoking [Rule Specificity](../PlusContent/RuleSpecificity.md).
```

Comment:
`every path is a Write[File/Key]Path`
to
`all drive paths are set to WriteFilePath`

Reply:
13257c3

Reply:
Fixed in the attached file. Thanks.
[privacy-mode.md](https://github.com/sandboxie-plus/sandboxie-docs/files/11971410/privacy-mode.md)